### ISSUE TITLE
Define prefixes in template header

### ISSUE THREAD

**Issue by white-gecko** [white-gecko/jekyll-rdf#1]
```

---
prefix.foaf: "<http://xmlns.com/foaf/0.1/>"
prefix.rdf: "<http://www.w3.org/1999/02/22-rdf-syntax-ns#>"

---

{{ page.rdf | rdf_property: 'foaf:job' }}
```

**Comment by white-gecko** [comment]
This should be fixed in #21

**Comment by white-gecko** [comment]
Done in #34
